src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/ReferenceDataflowSource.java

```diff
@@ -35,7 +34,13 @@
     public ReferenceDataflowSource(final PipelineOptions pipelineOptions, final String referenceURL,
                                    final SerializableFunction<GATKRead, SimpleInterval> referenceWindowFunction) {
         Utils.nonNull(referenceWindowFunction);
-        if (isFasta(referenceURL)) {
+        if (referenceURL.endsWith(".2bit")) {
+            try {
+                referenceSource = new ReferenceTwoBitSource(referenceURL, pipelineOptions);
+            } catch (IOException e) {
+                throw new IllegalStateException("Failed to create a ReferenceTwoBitSource object" + e.getMessage());
```

Comment:
Make this a `UserException` (as we generally blame problems with tool inputs on the user :)